src/lib/game/world/map.js

```diff
@@ -86,6 +92,10 @@ class Map extends THREE.Group {
           }
 
           this.add(m2);
+
+          if (this.world !== null && m2.billboards.length > 0) {
+            this.world.addBillboardedM2(m2);
+          }
```

Comment:
Multiple maps, phasing? I've heard about this before, but I never got that far into the game I think! :smile:

I think I'd prefer `update`, seeing as it might be used for something not for animation purposes, but a bit hard to say right now.